### DISCUSSION TITLE
Inappropriate quotes

### DISCUSSION THREAD

**Discussion by BenMCohen** [lukePeavey/quotable#155] [Quotable Data]
Not sure if we can block these types of quotes?
<img width="1334" alt="Screen Shot 2022-09-12 at 12 53 45 PM (1)" src="https://user-images.githubusercontent.com/114116253/191848644-25adce87-b01c-49d4-83dd-7e8229a43927.png">
![image](https://user-images.githubusercontent.com/114116253/191848820-75f42dc2-0a6d-4ea3-886a-ac231dca7079.png)

**Comment by lukePeavey** [comment]
The content guidelines for quotes are  

- No offensive language
- no quotes that might be interpreted as racist, sexist, or otherwise intolerant/offensive
- no quotes by historical figures who are associated with violence, racism, etc. 


The first quote that you mentioned should absolutely be removed, as it clearly violates the second guideline. Please report any similar quotes so that they can be removed as well.

The second quote is not offensive, but it is overtly political. In general, I have tried to avoid political quotes that are overtly partisan. I might consider replacing this with different Obama quote.

**Reply by lukePeavey** [reply]
Also just for future reference... 

The data for this API is managed in a separate repository. Any issues regarding content changes, such as adding or removing quotes, should be opened in the data repository 

https://github.com/quotable-io/data

**Reply by lukePeavey** [reply]
Sorry, disregard my last comment :)

**Comment by BenMCohen** [comment]
Thanks! And I'm for sure not offended by Obama. But it was for sure
political. Thanks again!

On Thu, Sep 22, 2022 at 8:52 PM Luke Peavey ***@***.***>
wrote:

> The content guidelines for quotes are
>
>    - No offensive language
>    - no quotes that might be interpreted as racist, sexist, or otherwise
>    intolerant/offensive
>    - no quotes by historical figures who are associated with violence,
>    racism, etc.
>
> The first quote that you mentioned should absolutely be removed, as it
> clearly violates the second guideline. Please report any similar quotes so
> that they can be removed as well.
>
> The second quote is not offensive, but it is overtly political. In
> general, I have tried to avoid political quotes that are overtly partisan.
> I might consider replacing this with different Obama quote.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/lukePeavey/quotable/discussions/155#discussioncomment-3713957>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A3GUNHN4UVWVZGDB7KNA2ZTV7USPFANCNFSM6AAAAAAQTNQHYQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 

Ben Cohen (he/him)

Product Management | 415.652.1276

<https://livelyme.com/>   <https://www.linkedin.com/in/benjaminmilancohen/>
<https://twitter.com/livelyhsa> <https://www.facebook.com/livelyhsa>

**Reply by lukePeavey** [reply]
I agree, I am going to swap that one out for a different quote. 

Please feel free to report any other you think should be removed

**Comment by lukePeavey** [comment]
@BenMCohen  I had a thought regarding the Obama quote... we could shorten this quote and just keep the first sentence: 

"No party has a monopoly on wisdom. No democracy works without compromise"
-  Barack Obama 

This is a nice quote, and is political without being overtly partisan. Let me know what you think.

**Comment by BenMCohen** [comment]
Yes! I love the first part of that quote!Sent from my iPhoneOn Sep 24, 2022, at 6:39 PM, Luke Peavey ***@***.***> wrote:﻿
@BenMCohen  I had a thought regarding the Obama quote... we could shorten this quote and just keep the first sentence:
"No party has a monopoly on wisdom. No democracy works without compromise"

Barack Obama

This is a nice quote, and is political without being overtly partisan. Let me know what you think.

**Comment by lukePeavey** [comment]
@BenMCohen this is done.
